library/common-library/templates/_labels.tpl

```diff
@@ -10,27 +10,30 @@ This function allows easily to add more labels to the function "common.labels"
 This will render the labels that should be used in all the manifests used by the helm chart.
 */}}
 {{- define "common.labels" -}}
-helm.sh/chart: {{ include "common.naming.chart" . }}
-{{ include "common.labels.selectorLabels" . }}
-{{- if .Chart.AppVersion }}
-app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
-{{- end }}
-app.kubernetes.io/managed-by: {{ .Release.Service }}
-
 {{- $global := index .Values "global" | default dict -}}
 
-{{- with $global.labels }}
-{{ toYaml . }}
-{{- end }}
+{{- $labels := dict "helm.sh/chart" (include "common.naming.chart" . ) -}}
+{{- $labels = mustMergeOverwrite $labels (fromYaml (include "common.labels.selectorLabels" . )) -}}
+{{- $labels = mustMergeOverwrite $labels (dict "app.kubernetes.io/managed-by" .Release.Service) -}}
```

Comment:
```suggestion
{{- $labels := dict "helm.sh/chart" (include "common.naming.chart" . ) -}}
{{- $selectorLabels = fromYaml (include "common.labels.selectorLabels" . ) -}}

{{- $labels = mustMergeOverwrite $labels $selectorLabels -}}
{{- $labels = mustMergeOverwrite $labels (dict "app.kubernetes.io/managed-by" .Release.Service) -}}
```

Reply:
Done in [#ca7876b](https://github.com/newrelic/helm-charts/pull/767/commits/ca7876be0239ef6f9f93c71e9a88a0481efba545)